.changeset/stale-wasps-hammer.md

```diff
@@ -0,0 +1,6 @@
+---
+"@cloudoperators/juno-app-heureka": minor
+"@cloudoperators/juno-app-greenhouse": patch
```

Comment:
Why is greenhouse included in the changeset? I see no changes to the greenhouse app in this PR

Reply:
My understanding is that if I want to deliver these changes into the greenhouse dashboard, I have to include greenhouse in my changeset. Isn't the case?